Application/Sources/UI/Controllers/PageContainerViewController.swift

```diff
@@ -51,99 +47,42 @@ class PageContainerViewController: UIViewController {
+        barView.buttons.customize { button in
+            button.contentMode = .scaleAspectFit
+            button.imageContentMode = .scaleAspectFit
+        }
+    }
+    
     override func loadView() {
         view = UIView(frame: UIScreen.main.bounds)
         view.backgroundColor = .srgGray16
         
-        let pageView = pageViewController.view!
-        view.insertSubview(pageView, at: 0)
-        
-        pageView.translatesAutoresizingMaskIntoConstraints = false
-        NSLayoutConstraint.activate([
-            pageView.topAnchor.constraint(equalTo: view.topAnchor),
-            pageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-            pageView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            pageView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
-        ])
-        
-        pageViewController.didMove(toParent: self)
-        
-        var hasImage = false
-        var tabBarItems = [UITabBarItem]()
-        viewControllers.forEach { viewController in
-            if let tabBarItem = viewController.tabBarItem {
-                tabBarItems.append(tabBarItem)
-                if tabBarItem.image != nil {
-                    hasImage = true
-                }
-            }
-        }
-        
-        let tabBar = MDCTabBar()
-        tabBar.itemAppearance = hasImage ? .images : .titles
-        tabBar.alignment = .center
-        tabBar.delegate = self
-        tabBar.items = tabBarItems
+        let tabView = tabContainerViewController.view!
+        view.insertSubview(tabView, at: 0)
         
-        tabBar.tintColor = .white
-        tabBar.unselectedItemTintColor = .srgGray96
-        tabBar.selectedItemTintColor = .white
-        
-        // Use ripple effect without color, so that there is no Material-like highlighting (we are NOT adopting Material)
-        tabBar.enableRippleBehavior = true
-        tabBar.rippleColor = .clear
-        
-        view.addSubview(tabBar)
-        self.tabBar = tabBar
-        
-        tabBar.translatesAutoresizingMaskIntoConstraints = false
-        tabBarTopConstraint = tabBar.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
-        NSLayoutConstraint.activate([
-            tabBarTopConstraint,
-            tabBar.heightAnchor.constraint(equalToConstant: 60.0),
-            tabBar.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            tabBar.trailingAnchor.constraint(equalTo: view.trailingAnchor)
-        ])
-        
-        let blurView = UIVisualEffectView.play_blurView
-        blurView.alpha = 0.0
-        view.insertSubview(blurView, belowSubview: tabBar)
-        self.blurView = blurView
-        
-        blurView.translatesAutoresizingMaskIntoConstraints = false
+        tabView.translatesAutoresizingMaskIntoConstraints = false
+        tabBarTopConstraint = tabView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
```

Comment:
```suggestion
        let tabBarTopConstraint = tabView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
        
        tabView.translatesAutoresizingMaskIntoConstraints = false
```

Declare contraint to avoid unwrap letter.

Reply:
I other code classes, it was more
1. declare
2. ise in a constraint array.
3. assign to a variable.

So I tried to share it in two conversation :) It was the next one. Sorry